R/SpatialExperiment.R

```diff
@@ -156,11 +154,19 @@
 #'     sample_id = "foo"))
 #'     
 #' #############################################################
-#' # Example 2: Spot-based ST (10x Visium) using 'read10xVisium'
+#' # Example 2: Spot-based ST (10x Visium) using 'VisiumIO'
 #' #############################################################
 #' 
-#' # see ?read10xVisium for details
-#' example(read10xVisium)
+#' library(VisiumIO)
```

Comment:
May I suggest to not depend on VisiumIO? 
I'd suggest it with a message, but it would be better to drop dependencies instead to add a new one

Reply:
Agree, that’s why it’s under suggests, so we are not depending on it actually

Reply:
ok, then!

Reply:
Yes, having it in `Suggests` is good!